### DISCUSSION TITLE
Why is there no `MulAssign` for `Affine2`?

### DISCUSSION THREAD

**Discussion by terrybrash** [bitshifter/glam-rs#420] [Q&A] [answered]
`Mul` is defined but not `MulAssign`. Is this intentional?

**Comment by terrybrash** [comment]
Is there any reason I shouldn't implement it and submit a PR?

**Reply by bitshifter** [reply, marked as answer]
Yeah I'm not sure why, probably forgot to add them. Feel free to create a pr, note that the affine files are generated, see https://github.com/bitshifter/glam-rs/blob/main/codegen/README.md
